compose/services/docker-compose.fuelrod-new.yml

```
@@ -4,26 +4,27 @@
+      - PHP_FPM_PM_MAX_CHILDREN=${FPM_MAX_CHILDREN:-10}
+      - PHP_FPM_PM_START_SERVERS=${FPM_START_SERVERS:-2}
+      - PHP_FPM_PM_MIN_SPARE_SERVERS=${FPM_MIN_SPARE:-1}
+      - PHP_FPM_PM_MAX_SPARE_SERVERS=${FPM_MAX_SPARE:-3}
     volumes:
     # - ${PWD}/:/var/www/html
     # - /var/www/html/node_modules
     # - /var/www/html/vendor
-    - ${PWD}/log/laravel:/var/www/html/storage/logs
-    - ${PWD}/log/supervisor:/var/log/supervisor
+    # - ${PWD}/tmp:/var/lib/nginx/tmp
+    - ${PWD}/log/nginx:/var/log/nginx
```

yamllint (1.35.1)

[error] wrong indentation: expected 6 but found 4

(indentation)